batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/controller/EmployeeController.java

```diff
@@ -0,0 +1,167 @@
+
+        else {
+            fetchData(request, response);
+        }
+
+    }
+
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        // TODO Auto-generated method stub
+        // doGet(request, response);
+        String path = request.getPathInfo();
+        if (path != null) {
+            System.out.println(path);
+            String[] pathParts = path.split("/");
+            if (pathParts[1].equals("createProcess")) {
+                createProcess(request, response);
+            } else if (pathParts[2].equals("editProcess")) {
+                int id = Integer.parseInt(pathParts[1]);
+                editProcess(request, response, id);
+            }
+        }
+
+    }
+
+    private void edit(HttpServletRequest request, HttpServletResponse response, int id) {
+        // TODO Auto-generated method stub
+        try {
+            emp = employeeService.fetchById(id);
+            request.setAttribute("employee", emp);
+            RequestDispatcher dispatcher = request.getRequestDispatcher("/WEB-INF/views/editEmployee.jsp");
+            dispatcher.forward(request, response);
+
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+    }
+
+    private void editProcess(HttpServletRequest request, HttpServletResponse response, int id) {
+        // TODO Auto-generated method stub
+        try {
+            System.out.println("done");
+            emp.setFirstName(request.getParameter("firstName"));
+            emp.setLastName(request.getParameter("lastName"));
+            emp.setDepartment(request.getParameter("department"));
+            emp.setAddress(request.getParameter("address"));
+
+            employeeService.edit(emp, id);
+
+            response.sendRedirect("/batch7crud-roll5/employees");
+
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+
+        }
+
+    }
+
+    private void create(HttpServletRequest request, HttpServletResponse response) {
+        // TODO Auto-generated method stub
+        try {
+            RequestDispatcher dispatcher = request.getRequestDispatcher("/WEB-INF/views/createEmployee.jsp");
+            dispatcher.forward(request, response);
+
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+
+    }
+
+    private void createProcess(HttpServletRequest request, HttpServletResponse response) {
+        // TODO Auto-generated method stub
+        try {
+            emp.setFirstName(request.getParameter("firstName"));
+            emp.setLastName(request.getParameter("lastName"));
+            emp.setDepartment(request.getParameter("department"));
+            emp.setAddress(request.getParameter("address"));
+
+            boolean status = employeeService.create(emp);
+
+            if (status == true) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "true" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1125)